### PR TITLE
add cards from VJMP-JP, AGOV-EN, & TW01-JP

### PR DESCRIPTION
**Pull request by Lyris12**
+ `Pendulum Witch`
+ `Play of the Tistina`

### CODE REVIEW

**Comment by Lyris12** [comment]
+ `Wattsychic Fighting Porter`

**Comment by Lyris12** [comment]
+ `Tainted of the Tistina`

**Comment by Lyris12** [comment]
+ `Returned of the Tistina`

File: scripts/AGOV-EN/c101202085.lua [hunk 1]
```diff
@@ -0,0 +1,51 @@
+--coded by Lyris
+--Wattsychic Fighting Porter
```
Comment (pokeboyexn) [inline scripts/AGOV-EN/c101202085.lua:2]: This one is committed under https://github.com/Fluorohydride/ygopro-pre-script/pull/1187

Comment (pokeboyexn) [reply scripts/AGOV-EN/c101202085.lua:2]: You submitted before him. Please ignore the above comment

**Comment by Lyris12** [comment]
+ `R-Genex Undine`

File: scripts/VJMP-JP/c100200244.lua [hunk 2]
```diff
@@ -0,0 +1,116 @@
...
+	e2:SetCategory(CATEGORY_DESTROY+CATEGORY_SEARCH+CATEGORY_TOHAND)
+	e2:SetType(EFFECT_TYPE_SINGLE+EFFECT_TYPE_TRIGGER_O)
+	e2:SetCode(EVENT_SUMMON_SUCCESS)
+	e2:SetCountLimit(1,id+o)
+	e2:SetProperty(EFFECT_FLAG_CARD_TARGET+EFFECT_FLAG_DELAY)
+	e2:SetTarget(s.thtg)
+	e2:SetOperation(s.thop)
+	c:RegisterEffect(e2)
+	local e3=e2:Clone()
+	e3:SetCode(EVENT_SPSUMMON)
```
Comment (Crescent0v0) [inline scripts/VJMP-JP/c100200244.lua:32]: fix parameter
```suggestion
	e3:SetCode(EVENT_SPSUMMON_SUCCESS)
```

**Pull request merged**